csharp/sources/Valkey.Glide/Commands/IHashCommands.cs

```diff
@@ -1,56 +1,253 @@
-
-    Task<ExpireResult[]> HashFieldExpireAsync(ValkeyKey key, ValkeyValue[] hashFields, DateTime expiry, ExpireWhen when = ExpireWhen.Always, CommandFlags flags = CommandFlags.None);
-
-    Task<long[]> HashFieldGetExpireDateTimeAsync(ValkeyKey key, ValkeyValue[] hashFields, CommandFlags flags = CommandFlags.None);
-
-    Task<PersistResult[]> HashFieldPersistAsync(ValkeyKey key, ValkeyValue[] hashFields, CommandFlags flags = CommandFlags.None);
-
-    Task<long[]> HashFieldGetTimeToLiveAsync(ValkeyKey key, ValkeyValue[] hashFields, CommandFlags flags = CommandFlags.None);
-
-    Task<ValkeyValue> HashGetAsync(ValkeyKey key, ValkeyValue hashField, CommandFlags flags = CommandFlags.None);
-
-    //Task<Lease<byte>?> HashGetLeaseAsync(ValkeyKey key, ValkeyValue hashField, CommandFlags flags = CommandFlags.None);
-
-    Task<ValkeyValue[]> HashGetAsync(ValkeyKey key, ValkeyValue[] hashFields, CommandFlags flags = CommandFlags.None);
-
-    Task<HashEntry[]> HashGetAllAsync(ValkeyKey key, CommandFlags flags = CommandFlags.None);
-
-    Task<long> HashIncrementAsync(ValkeyKey key, ValkeyValue hashField, long value = 1, CommandFlags flags = CommandFlags.None);
-
-    Task<double> HashIncrementAsync(ValkeyKey key, ValkeyValue hashField, double value, CommandFlags flags = CommandFlags.None);
-
-    Task<ValkeyValue[]> HashKeysAsync(ValkeyKey key, CommandFlags flags = CommandFlags.None);
-
-    Task<long> HashLengthAsync(ValkeyKey key, CommandFlags flags = CommandFlags.None);
-
-    Task<ValkeyValue> HashRandomFieldAsync(ValkeyKey key, CommandFlags flags = CommandFlags.None);
-
-    Task<ValkeyValue[]> HashRandomFieldsAsync(ValkeyKey key, long count, CommandFlags flags = CommandFlags.None);
-
-    Task<HashEntry[]> HashRandomFieldsWithValuesAsync(ValkeyKey key, long count, CommandFlags flags = CommandFlags.None);
-
-    Task HashSetAsync(ValkeyKey key, HashEntry[] hashFields, CommandFlags flags = CommandFlags.None);
-
-    Task<bool> HashSetAsync(ValkeyKey key, ValkeyValue hashField, ValkeyValue value, When when = When.Always, CommandFlags flags = CommandFlags.None);
-
-    Task<long> HashStringLengthAsync(ValkeyKey key, ValkeyValue hashField, CommandFlags flags = CommandFlags.None);
-
-    Task<ValkeyValue[]> HashValuesAsync(ValkeyKey key, CommandFlags flags = CommandFlags.None);
 
+/// <summary>
+/// Supports commands for the "Hash Commands" group for standalone and cluster clients.
+/// <br/>
+/// See more on <see href="https://valkey.io/commands/?group=hash#hash">valkey.io</see>.
+/// </summary>
+public interface IHashCommands
+{
+    /// <summary>
+    /// Returns the value associated with field in the hash stored at key.
+    /// </summary>
+    /// <param name="key">The key of the hash.</param>
+    /// <param name="hashField">The field in the hash to get.</param>
+    /// <param name="ignored">The flags to use for this operation. Currently flags are ignored.</param>
```

Comment:
As discussed, we cannot rename arguments, because a user may use named arguments. Have to keep the original names.